doc/lua_api.txt

```diff
@@ -3777,6 +3777,23 @@ The coordinates are *inclusive*, like most other things in Minetest.
       `[z [y [x]]]`.
 * `iterp(minp, maxp)`: same as above, except takes a vector
 
+### Y stride and z stride of a flat array
+
+If, for example:
+
+    local area = VoxelArea:new{MinEdge = emin, MaxEdge = emax}
```

Comment:
I think the example of creating an area is unnecessary here

Reply:
Describing these as 'properties' of a VoxelArea is not clear,even to me. The values are got by '.ystride' after a variable, so that variable must be stated as an example. This is the only way i could understand it myself, so imagine how it will be for others =)